doc/modules/ensemble.rst

```diff
@@ -1071,3 +1071,207 @@ must support ``predict_proba`` method)::
 Optionally, weights can be provided for the individual classifiers::
 
    >>> eclf = VotingClassifier(estimators=[('lr', clf1), ('rf', clf2), ('gnb', clf3)], voting='soft', weights=[2,5,1])
+
+
+.. _stacking:
+
+Stacked generalization
+======================
+
+.. currentmodule:: sklearn.ensemble
+
+Stacked generalization is another method of combining estimators to reduce
+their biases [W1992]_ by combining several estimators (possibly non-linearly)
+stacked together in layers. Each layer will contain estimators and their
+predictions are used as features to the next layer.
+
+Considering one of the layers in a stacked generalization, the method is as
```

Comment:
As this is not what is implemented in `make_stack_layer`, I would move this description in the section called "Stacked generalization without cross validation".

Instead, the cross-validation implemented in `make_stack_layer` should be properly described (which is currently not the case).